app/facades/user_books_facade.rb

```diff
@@ -1,7 +1,32 @@
 class UserBooksFacade
 
-  def self.all_books
-    
+  def self.all_books(auth_token)
+    shelves = BookService.book_shelves(auth_token)
+    shelf_ids = shelves[:items].map do |shelf|
+      if shelf[:title] == "Reading now" || shelf[:title] == "To read" || shelf[:title] == "Have read"
```

Comment:
Perhaps a refactor here could be to put all accepted names in an array and use `.include?(shelf[:title])` - this may make it easier to add potential shelves down the road without adding too much more logic!